gubernator.go

```diff
@@ -112,6 +112,16 @@ func (s *Instance) GetRateLimits(ctx context.Context, r *GetRateLimitsReq) (*Get
 					return nil
 				}
 
+				var attempts int
+			getPeer:
+				if attempts > 5 {
```

Comment:
What increments `attempts`, could the `goto` on line 157 cause an infinite loop?

Reply:
DOH! (This is why we have code reviews!) Will fix!